### PR TITLE
Update to latest IBMQuantumExperience interface

### PR DESCRIPTION
**Pull request by RolfHuisman**
Upgrade to the latest interface (which doesn't always give an expected queue size).

### CODE REVIEW

**Comment by RolfHuisman** [comment]
Simplified the code to just wait for the timeout. Much simpler and more reliable. Sometimes the environment is fast enough that this just works.

File: Samples/Qiskit/Driver.cs [hunk 1]
```diff
@@ -22,14 +22,16 @@ static void Main(string[] args)
...
-            var factory = new IbmQx4(apiKey); //Using different Factory
-            Console.WriteLine("Hadamard on IBMQx4");
-            for (int i = 0; i < 1; i++)
+            else
             {
-                var result = MeasurementOneQubit.Run(factory).Result;
-                Console.WriteLine($"Result of Hadamard is {result}");
+                var factory = new IbmQx4(apiKey); //Using different Factory
+                Console.WriteLine("Hadamard on IBMQx4");
+                for (int i = 0; i < 1; i++)
```
Comment (anpaz-msft) [inline Samples/Qiskit/Driver.cs]: ?

File: Samples/Qiskit/Qiskit.csproj [hunk 2]
```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Content Include="QiskitInterface.py">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (anpaz-msft) [inline Samples/Qiskit/Qiskit.csproj]: ?

Comment (RolfHuisman) [reply Samples/Qiskit/Qiskit.csproj]: Accidental inclusion. Will correct.

File: Samples/Qiskit/Driver.cs (hunk 1 above)
Comment (RolfHuisman) [reply Samples/Qiskit/Driver.cs]: Moved the if statement, so it halts after the message. Otherwise one wouldn't see it.
The diff, makes it look wierd.

**Comment by kok11** [comment]
Ok

**Comment by RolfHuisman** [comment]
@kok11 thanks.

**Comment by anpaz-msft** [comment]
Somehow I missed this had been updated. Sorry for the delay.

**Pull request merged**